packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -66,6 +66,8 @@ const ReactDataGrid = React.createClass({
     onCellsDragged: React.PropTypes.func,
     onAddFilter: React.PropTypes.func,
     onGridSort: React.PropTypes.func,
+    sortColumn: React.PropTypes.string,
+    sortDirection: React.PropTypes.oneOf(['ASC', 'DESC', 'NONE']),
```

Comment:
I agree. Maybe we can move these definitions to `Constants.js` or `AppConstants.js` file?